.plugin-manifests/generate-manifest.sh

```diff
@@ -16,7 +16,7 @@ PLUGIN_BINARY_VERSION_STRING=$1
 
 # If canary release tag with epoch at the end as it is monotonic
 if [[ $VERSION == "canary" ]]; then
-    PLUGIN_VERSION=$(cat ./Cargo.toml | grep version | head -n 1 | awk '{print $3}')
+    PLUGIN_VERSION=$(cat ./Cargo.toml | grep version | head -n 2 | tail -n 1 | awk '{print $3}')
```

Comment:
If you can depend on `jq`:

```suggestion
    PLUGIN_VERSION=$(cargo read-manifest | jq -r .version)
```

Reply:
Ohh Yes, that is much cleaner! Thanks